src/opentelemetry/lightstep/configuration.py

```diff
@@ -88,10 +104,10 @@ def configure_opentelemetry(
         access_token (str): LS_ACCESS_TOKEN, the access token used to
             authenticate with the Lightstep satellite. This configuration value
             is mandatory.
-        satellite_url (str): LS_SATELLITE_URL, the URL of the Lightstep
+        satellite_url (str): OTEL_EXPORTER_OTLP_SPAN_ENDPOINT, the URL of the Lightstep
```

Comment:
i'd update the variable name here, it's confusing otherwise :)

Reply:
Fixed